docs/platform/howto/manage-payment-card.rst

```diff
@@ -1,14 +1,14 @@
-Manage payment card 
+Manage payment cards 
 ======================
-For a project using credit card billing, a payment card is the credit card to which all project billing is invoiced. The payment card, also known as the project's credit, is owned by the billing contact. 
-The billing contact can add or update the payment card and assign project(s) to this payment card. 
+You can a add payment card to your organization and use it as the payment method for one or more billing groups.
 
 
-Add or update payment card
-~~~~~~~~~~~~~~~~~~~~~~~~~~
-The billing contact can add or update the payment card and assign project(s) to this payment card. 
+Add a payment card
+~~~~~~~~~~~~~~~~~~~~
+To add a payment card in the `Aiven Console <https://console.aiven.io/>`_: 
 
-1. In the `Aiven web console <https://console.aiven.io/>`_, click the **User information** icon on the top right of the screen to view the User profile. 
-2. Next, click the **Payment Options** tab.
-3. You can add a new payment card or update the existing payment card, in the **Payment Options** tab. 
-4. If you have added a new payment card, ensure to assign this card to project(s). 
+#. Click **Billing**. 
+#. On the **Payment cards** tab in the **Add payment card** section, enter the credit card details. 
```

Comment:
There is one step between. Billing -> Payment methods -> Payment cards. Payment cards is currently the only tab within payment methods.